code/modules/surgery/organs_internal.dm

```diff
@@ -36,6 +36,17 @@ and organ transplant code which may come in handy in future but haven't been edi
 	)
 	time = 3 SECONDS
 	repeat_step = TRUE
+	///How many stacks to use for each organ.
+	var/use_stack = 1
+
+// Use materials to repair organs
+/datum/surgery_step/repair_organs/extra_checks(mob/living/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery, repeating, skipped)
```

Comment:
This eats a stack before you complete the surgery, meaning if you drop the item, move away or switch hands you will just delete one for no reason, while you refunded one below at a failing step.